wagtail/core/models.py

```diff
@@ -2287,6 +2287,8 @@ class Collection(MP_Node):
     name = models.CharField(max_length=255, verbose_name=_('name'))
 
     objects = CollectionManager()
+    # Tell treebeard to order Collections' paths such that they are also ordered by name at the top level.
+    node_order_by = ['name']
```

Comment:
`test_get_prev_siblings` needs updating to reflect the order change

Reply:
Oh duh! I added that `node_order_by` change at the last minute, and completely forgot to run the usual checks afterward. Will fix.

Reply:
My my, that was an interesting challenge. I fixed up the `TestCollectionTreeOperations` tests to work properly with the new sorting, but they still weren't working. It didn't make any sense, until I realized that by setting `Collection.node_order_by`, we've configured treebeard to potentially change the paths of Collections in the DB without updating any existing Collection objects in memory. That's why the use of `refresh_from_db()` became necessary.

I also corrected the no-longer-accurate sorting tests in the `TestCollectionsIndexView` tests, and corrected an error in the Collection Edit view as well (you can't use `last-child` as a Collection move target any more, because `node_order_by` sorts the the new Collection for you, whether you want it to be last or not).